### ISSUE TITLE
Enhance logging to show the current origin/device for every log line if related

### ISSUE THREAD

**Issue by sn0opy** [Map-A-Droid/MAD#736]
Right, now. The device name either randomly somewhere in the log line or not included at all eventhough it should be in there because it is actually related.

Lines like these are okay:

```
[02-06 12:51:10.93] [       device007] [         WorkerBase:568 ] [    INFO] turnScreenOnAndStartPogo: (Re-)Starting Pogo
```

not okay:

```
[02-06 12:51:41.19] [          scanner] [      WorkerFactory:120 ] [    INFO] device007 using walker area my_area_name [19/23]
```

I propose the following change:

* get rid of Python class + line number (or display them only in debug mode)
* replace the thread name with the actual device name and use "system" if it's a general log line and not related to a device

The latter is probably the most annyoing thing, depending on the where and how it's set. Some other thread names are named to contain the device name and the "type" like "worker_device007" which does not work for all places in the code.

Loguru provides a handy way to add custom fields to log lines via `.bind()`.
https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.bind

**Comment by Grennith** [comment]
I would prefer to leave the `class:line` be, it does help reading and filtering debug logs as to where to start and look next in a sequence of calls.
The same applies for the threads - we could use some cleanup/clarification there tho!

In general, it would be a good idea to add the origin/device name to every logline related to a worker however

**Comment by sn0opy** [comment]
> I would prefer to leave the `class:line` be, it does help reading and filtering debug logs as to where to start and look next in a sequence of calls.
> The same applies for the threads - we could use some cleanup/clarification there tho!

It definetely does. An option for advanced users allowing them to hide this would be nice then I guess.

**Comment by Expl0dingBanana** [comment]
This was implemented as part of #924
